scripts/install_third_party.sh

```diff
@@ -117,6 +117,25 @@ fi
 echo Checking if numpy is installed in $TOOLS_DIR/pip_packages
 if [ ! -d "$TOOLS_DIR/numpy-1.6.1" ]; then
   echo Installing numpy
+
+  if ! type pip > /dev/null 2>&1 ; then
```

Comment:
I don't think so, since `type` is a command not an expression, I tested this and it works this way